src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class Button extends Component {
+    render(){
+        return <button onClick={this.props.bookingsById} className="btn btn-primary fn-submit-name">{this.props.name}</button>;
```

Comment:
This is subjective. But I don't see the point of separating Button into its own component, it should just be inside Search component. It is not a generic button, and it only does one thing related to the search functionality.

Reply:
I don't know, it was on the task to create a separate component for a button. I think it's just to show us how to create new component, because at that time we had no idea about React components.